script/suspend_inactive.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Run it with rails runner, for example:
+# $ bundle exec rails runner suspend_inactive.rb
+
+require 'progress_counter'
+
+module SuspendInactiveProviders
+
+  module_function
+
+  PLANS = [
+    2357355454121, # 2013 Enterprise (Trial)
+    2357355852196, # 90 Day Trial (50K calls/day)
+    2357355916595, # Enterprise (Eval)
+    2357355814696, # Developer
+    2357355852194, # Personal (5K calls/day)
+  ]
+
+  BATCH_SIZE = ENV.fetch("BATCH_SIZE", 50).to_i
```

Comment:
I think the name `BATCH_SIZE` is not appropriate, as it suggests that still all the accounts will be processed, just in batches of (in this case) 50.

I guess a more suitable name for the current functionality would be just `LIMIT`, because this is what it does.
But I understand that we're not going to merge this PR, right? in this case it's not that important.